index.html

```diff
@@ -22,8 +22,25 @@
   gtag('config', 'G-7960NHDJR9');
 </script>
 </head>
-
 <body>
+    <div class="blur">
+        <div class="mobile">
+            <ul class="menuMenu">
+                    <li class="nav-tag">
+                    <a class="portfolio"  href="#portfolio-sec">Portfolio</a>
+                    </li>
+                    <li class="nav-tag">
+                    <a  href="#about-sec">About</a>
+                    </li>
+                    <li class="nav-tag">
+                    <a  href="#contact-sec">Contact</a>
+                    </li>
+            </ul>
+            <button type="button" class ="hamburger">
+                <i class="menuIcon fas fa-bars"> </i>
+                <i class="closeIcon fas fa-times"></i>
+            </button>
```

Comment:
The _mobile menu_ **does not appear/popup** when the _hamburger_ is clicked/pressed. Kindly ensure to improve upon the following:
- [ ] When the user clicks (or taps) the hamburger button, the mobile menu appears.
- [ ] When the user clicks (or taps) the close (X) button, the mobile menu disappears.
- [ ] When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.
- [ ] When the user clicks (or taps) any option of the mobile menu, the mobile menu disappears.

Reply:
Mobile menu works as expected: